### DISCUSSION TITLE
Galaxy Brain Achievement Need Help

### DISCUSSION THREAD

**Discussion by nindaanisa** [Schweinepriester/github-profile-achievements#386] [Q&A] [answered]
Here!

**Comment by radespratama** [comment, marked as answer]
to achieve the 'Galaxy Brain' achievement, take note of the following:

- Answer at least 2 discussions.
- Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user.
- Discussions should be in a public repository.
- There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.

Hope this answer has helped you, and all the best in getting the achievement! :)

Reference : ![#18467](https://github.com/orgs/community/discussions/18467)

**Reply by miladkhalighi** [reply]
Up

**Comment by miladkhalighi** [comment]
Here
